src/models/Chain.js

```diff
@@ -36,15 +36,9 @@ export type ChainRecord<T> = {|
 
 // Based on: https://chainid.network/
 export const CHAIN_ID = {
-  ETHEREUM: 1,
+  ETHEREUM_MAINNET: 1,
+  ETHEREUM_KOVAN: 42,
   POLYGON: 137,
   BINANCE: 56,
   XDAI: 100,
```

Comment:
Could we add `FANTOM: 250` here too, just incase we support that in the app too in future? That way we don't have to come back and change this.